pkg/apitype/policy.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apitype
+
+// CreatePolicyPackRequest defines the request body for creating a new Policy
+// Pack for an organization. The request contains the metadata related to the
+// Policy Pack.
+type CreatePolicyPackRequest struct {
+	Name        string   `json:"name"`
+	Version     int      `json:"version"`
```

Comment:
I'm a little surprise to see `version` here.  I would have expected the `version` value to be assigned as part of processing the request (so we can assign the latest version) and then it returned as part of the response.

Reply:
Yes that is the plan!

Good catch!

Reply:
I am actually not quite sure I understand why having the server assign a version is better than having the user assign a semver string, and having the server map that to whatever representation it wants on the backend.

Reply:
Bc then we need to enforce some sort of versioning for the semver, which complicates things. Dealing w 1 number if easier that than three numbers that are a really a string.